tests/pages/update-address-page.test.tsx

```diff
@@ -0,0 +1,64 @@
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { SWRConfig } from "swr";
+
+import { render, screen } from "../config/testing-library-config";
+import * as mockAddresses from "../mocks/addresses";
+import EditAddressPage from "@src/pages/account/profile/addresses/[id]/edit";
+
+jest.mock("@src/configs/db-client", () => {
+  return {
+    dynamoDbClient: {
+      send: () => {
+        return;
+      },
+    },
+  };
+});
+
+describe("EditAddressPage", () => {
+  it("should render proper user address form elements", async () => {
+    expect.assertions(20);
+
+    render(
+      <SWRConfig
+        value={{
+          dedupingInterval: 0,
+          fetcher: () => {
+            return { address: mockAddresses.primaryAddress };
+          },
+        }}
+      >
+        <EditAddressPage addressId={mockAddresses.primaryAddress.id} />
+      </SWRConfig>,
+    );
+
+    expect(
+      screen.getByRole("heading", { name: /address \| edit/i }),
+    ).toBeInTheDocument();
+
+    const textInputs = await screen.findAllByRole("textbox");
+
+    expect(textInputs).toHaveLength(7);
```

Comment:
```suggestion
    expect(textInputs).toHaveLength(Object.keys(mockAddresses.primaryAddress).length);
```
This way, even if we're modifying the mock data, we won't have to change the related test

Reply:
The mocked data has more keys than the ones we are currently "displaying" a text input to update them. It is the same keys as the ones used in the form state (that correspond to what the GET endpoint send us back, since we didn't have cherrypick the keys we use for now, if I'm correct).